templates/individualModuleInfo.html

```diff
@@ -9,6 +9,12 @@
 $ title_string = 'Module Info for ' + moduleCode + ' in ' + aySem
 
 $var title:$title_string
+$ home = ['/', 'Home']
+$ moduleInfo = ['/modules', 'Module Information']
+$ viewModule = ['/viewModule?code=' + moduleCode, moduleCode]
+$ individualModuleInfo = ['#' + aySem, 'Module Info for ' + aySem]
```

Comment:
![image](https://cloud.githubusercontent.com/assets/12386860/24077764/d5f17036-0c92-11e7-99a8-4a4cec1f327a.png)

The last link in the breadcrumbs should not be clickable since it is <code>active</code>. Perhaps everything else after the '#' can be dropped out because your conditional checks `$if entry[0] != #` in `base.html`.